iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -202,8 +204,11 @@ public static Schema readSchema(Configuration conf) {
   }
 
   public static String[] selectedColumns(Configuration conf) {
-    String[] readColumns = conf.getStrings(InputFormatConfig.SELECTED_COLUMNS);
-    return readColumns != null && readColumns.length > 0 ? readColumns : null;
+    String readColumns = conf.get(InputFormatConfig.SELECTED_COLUMNS);
+    if (readColumns == null) {
```

Comment:
The original code returned `null` for empty `readColumns` too. Are we ok with this change?

Reply:
Nope, you're right. I've already fixed this on my local machine, will push it soon. This was the reason for the failed HADOOP_TABLE tests.